Source/JavaScriptCore/runtime/ISO8601.cpp

```diff
@@ -1552,6 +1552,16 @@ String temporalDateToString(PlainDate plainDate)
     return temporalDateToString(plainDate.year(), plainDate.month(), plainDate.day());
 }
 
+String temporalYearMonthToString(PlainYearMonth plainYearMonth, StringView calendarName)
+{
+    if (calendarName == "always"_s) {
+        // FIXME: print the correct calendar ID when calendars are fully implemented
+        auto first = temporalDateToString(plainYearMonth.isoPlainDate());
+        return makeString(first, "[u-ca=iso8601]"_s);
```

Comment:
Why the local variable?
```suggestion
        return makeString(temporalDateToString(plainYearMonth.isoPlainDate()), "[u-ca=iso8601]"_s);
```